lms/djangoapps/organizations/views.py

```diff
@@ -96,8 +106,13 @@ def users(self, request, pk):
             include_course_counts = request.QUERY_PARAMS.get('include_course_counts', None)
             include_grades = request.QUERY_PARAMS.get('include_grades', None)
             course_id = request.QUERY_PARAMS.get('course_id', None)
+            view = request.QUERY_PARAMS.get('view', None)
             grade_complete_match_range = getattr(settings, 'GRADEBOOK_GRADE_COMPLETE_PROFORMA_MATCH_RANGE', 0.01)
             course_key = None
+            if view == 'ids':
```

Comment:
As with my comment on 'groups' below, I think we should move this view formatter down to just before the main return statement, and we should break apart the query filter from the id value extractor.